kmip/core/config_helper.py

```diff
@@ -35,6 +35,7 @@ class ConfigHelper(object):
     DEFAULT_SSL_VERSION = 'PROTOCOL_SSLv23'
     DEFAULT_USERNAME = None
     DEFAULT_PASSWORD = None
+    DEFAULT_TIMEOUT = 30
```

Comment:
Can you add a comment about the units for this? I assume this is seconds. What about milliseconds?